src/commons/big_query/big_query_job_error.py

```diff
@@ -9,6 +14,15 @@ def __str__(self):
         return "{} while creating Copy Job from {} to {}" \
             .format(self.reason, self.source_bq_table, self.target_bq_table)
 
+    def create_post_copy_action(self, copy_job_request):
```

Comment:
As discussed, lets consider using different name. Otherwise it suggests its polymorphic, method correlated with BigQueryJobReference, where in fact they're 2 different classes